mxfusion/inference/logger.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License").
+#   You may not use this file except in compliance with the License.
+#   A copy of the License is located at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   or in the "license" file accompanying this file. This file is distributed
+#   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+#   express or implied. See the License for the specific language governing
+#   permissions and limitations under the License.
+# ==============================================================================
+
+
+import os
+import time
+
+from mxboard import SummaryWriter
```

Comment:
Could we avoid the explicit dependency on mxboard?

Reply:
Would you prefer the import be wrapped in a try except with a warning?

```suggestion
try:
    from mxboard import SummaryWriter
except ImportError:
    warning('Could not import mxboard. TensorBoard logs will not be available.')
```

Reply:
The tensorboard feature will only be executed, if a user configures it. We can simply put the import inside the function that requires ```SummaryWriter```.